src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceIntTest_addresses.kt

```diff
@@ -139,5 +139,106 @@ class OffenderResourceIntTest_addressesIntTest : ResourceTest() {
         }
       }
     }
+
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    inner class HappyPath {
+      private lateinit var prisoner: OffenderId
+      private lateinit var prisonerNoAddresses: OffenderId
+
+      @BeforeAll
```

Comment:
having got my fingers bunt - write a proper test with multiple bookings and aliases (OFFENDER records)